packages/augur-ui/jest.config.js

```diff
@@ -67,11 +67,7 @@ module.exports = {
 
   // A map from regular expressions to module names that allow to stub out resources with a single module
   moduleNameMapper: {
-    "^src/(.*)$": "<rootDir>/src/$1",
-    "^(config|modules|utils|services)/(.*)$": "<rootDir>/src/$1/$2",
-    "^assertions/(.*)$": "<rootDir>/test/assertions/$1",
-    "^test/(.*)$": "<rootDir>/test/$1",
-    "\\.(style|styles)$": "<rootDir>/test/mockStyle.js"
+    "\\.styles?$": "<rootDir>/test/mockStyle.js"
```

Comment:
I think there might be include files that end with `.style` would that be an issue?

Reply:
That is just shorthand for both style and styles. It was done on another branch and shouldn't haven't show up here.

Reply:
Will rebase it out.